SalesforceCommerceCloud/pwa-kit
# Add tests for optional apiClient configs like headers and fetchOptions

# Description
Add tests for optional apiClient configs like headers and fetchOptions



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- src/provider.test.tsx

# How to Test-Drive This PR

- npm run test src/provider.test.tsx at packages/commerce-react-sdk package
- Observe all tests are passed

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
